coreos-base/update_engine/update_engine-9999.ebuild

```diff
@@ -8,7 +8,7 @@ CROS_WORKON_REPO="https://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="12d43a8e71293567f10d940465113550188a4ce8" # flatcar-master
+	CROS_WORKON_COMMIT="449f700e494c45d30e618925660a6b9237182152" # krnowak/curl
```

Comment:
That commit seems gone.
The correct one is actually `c6f566d47d8949632f7f43871eb8d5c625af3209` (flatcar-master).

Reply:
Oh yeah, now that I merged the PR for update_engine, I need to update it here too.

Reply:
Fixed.